src/azure-cli/az

```diff
@@ -1,14 +1,16 @@
-#!/usr/bin/env bash
+#!/usr/bin/env python
 
```

Comment:
Any earnings changing from bash to python?

Reply:
I listed the benefits when opening this pull request: https://github.com/Azure/azure-cli/pull/10516#issue-316767937

Does that answer your question, or did you mean something else by "earnings"?